<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quizzdesign</class>
 <widget class="QMainWindow" name="quizzdesign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>12</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>quizzdesign</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget
{
	background-color: white;

}

#TitreLabel
{
	color:#3366ff;
	font-family: verdana;
}

#JouerBtn, #OptionBtn,#QuitterBtn, #ValiderOptionBtn,#ValiderReponseBtn {
text-decoration: none;
padding: 11px;
font-family: verdana;
/*font-size: 12px;*/
color: #ffffff;
background-color: #3366ff;
border-radius: 10px ;

border: 4px solid #ffffff;


}


#JouerBtn::hover, #OptionBtn::hover, #QuitterBtn::hover, #ValiderOptionBtn::hover,#ValiderReponseBtn::hover 

{
padding: 10px;
font-family: verdana;
color: #3366ff;
background-color: #ffffff;
border: 4px solid #3366ff;
box-shadow: 1px 1px 4px #777777;
-webkit-box-shadow: 1px 1px 4px #777777;
-moz-box-shadow: 1px 1px 4px #777777;
}

#ValiderOptionBtn
{
	margin-top:8px;
	padding-top:3px;
}

#LabelOption
{
	color:#3366ff;
}

#LabelQuestion
{
	color:#3366ff;
	margin-bottom:60px
	
}

#ValiderReponseBtn
{
	margin-top: 20px;
	margin:30px;
}




</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="MenuPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QWidget" name="MenuContainer" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="TitreLabel">
             <property name="font">
              <font>
               <family>verdana</family>
               <pointsize>20</pointsize>
               <italic>true</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Qui veut gagner des pepette</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QPushButton" name="JouerBtn">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>verdana</family>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Jouer</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QPushButton" name="OptionBtn">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>verdana</family>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Option</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QPushButton" name="QuitterBtn">
             <property name="minimumSize">
              <size>
               <width>160</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>160</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>verdana</family>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Quitter</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="OptionPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="OptionContainer" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="OptionLabelContainer" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
               <widget class="QLabel" name="LabelOption">
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Option</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QWidget" name="OptionThemeContainer" native="true">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>20</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QWidget" name="OptionTheme3" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_10">
                 <property name="spacing">
                  <number>20</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QCheckBox" name="checkBoxLangue">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Langue</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBoxHistoire">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Histoire</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBoxCulture">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Culture</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="OptionTheme2" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <property name="spacing">
                  <number>20</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QCheckBox" name="checkBoxJv">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Jeux vidéo</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBoxMusique">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Musique</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBoxCine">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Cinéma</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="OptionTheme1" native="true">
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <property name="spacing">
                  <number>20</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QCheckBox" name="checkBoxMath">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Mathématiques</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBoxScience">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Sciences</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="checkBoxSport">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>12</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Sport</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
             <zorder>OptionTheme2</zorder>
             <zorder>OptionTheme3</zorder>
             <zorder>OptionTheme1</zorder>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QWidget" name="OptionNivContainer" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
               <widget class="QComboBox" name="OptionComboBox">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>Facile</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Difficile</string>
                 </property>
                </item>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QPushButton" name="ValiderOptionBtn">
                <property name="minimumSize">
                 <size>
                  <width>160</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>verdana</family>
                  <pointsize>11</pointsize>
                  <bold>true</bold>
                  <underline>false</underline>
                  <strikeout>false</strikeout>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>Valider</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="JeuPage">
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item alignment="Qt::AlignTop">
         <widget class="QWidget" name="HeaderContainer" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item alignment="Qt::AlignLeft|Qt::AlignTop">
            <widget class="QLabel" name="LabelQst">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Question:</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignTop">
            <widget class="QLabel" name="LabelNumeroQuestion">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>1/2</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignRight|Qt::AlignTop">
            <widget class="QLabel" name="LabelMontant">
             <property name="font">
              <font>
               <family>Verdana</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0€</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QWidget" name="BodyContainer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QWidget" name="BodyContentContainer" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QLabel" name="LabelQuestion">
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Combien font 2+2 ?</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="RadioBtnQ2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>485</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="RadioBtnQ1">
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>78</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="RadioBtnQ3">
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="RadioBtnQ4">
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>16</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>4</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="InvisibleRadioButton">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="ValiderReponseBtn">
                <property name="font">
                 <font>
                  <family>verdana</family>
                  <pointsize>16</pointsize>
                  <underline>false</underline>
                  <strikeout>false</strikeout>
                 </font>
                </property>
                <property name="text">
                 <string>Valider</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
